pgxn/neon/libpagestore.c

```diff
@@ -527,37 +699,48 @@ pageserver_disconnect(shardno_t shard_no)
 static void
```

Comment:
The comment above, in `pageserver_disconnect`, probably should be adjusted: we don't check `page_servers[shard_no].conn == NULL` anymore